services/core-api/app/api/mines/notice_of_departure/resources/notice_of_departure.py

```diff
@@ -51,7 +51,7 @@ def patch(self, mine_guid, nod_guid):
 
         nod.nod_title = data.get('nod_title')
         nod.nod_description = data.get('nod_description')
-        nod.nod_type = NodType[data.get('nod_type', 'pending_review')]
+        nod.nod_type = NodType[data.get('nod_type')]
         nod.nod_status = NodStatus.pending_review
```

Comment:
This is going to set nod to pending_review always.

Reply:
Yes, that's how it had already been set.  Will be addressing that in the upcoming PR.